src/Handler/Project/Create/CreateProjectHandler.php

```diff
@@ -121,6 +121,13 @@ public function __invoke(
             TeradataQuote::quoteSingleIdentifier($newProjectUsername)
         ));
 
+        // grant crete/drop role to project user
+        // this is needed to create workspace
+        $db->executeStatement(sprintf(
+            'GRANT CREATE ROLE, DROP ROLE TO %s WITH GRANT OPTION;',
```

Comment:
ten grant option tady není potřeba ne? On vytváří WS uživatele a roli, ale už ten grant nepředává ne?

Reply:
Nebyl by špatný test, že ten grant má.

Reply:
je to potreba, vytravim tady roli pro workspace:
```proto
message CreateWorkspaceResponse {
  string workspaceUserName = 1; // newly created user name associated with workspace
  string workspaceRoleName = 2; // newly created role name associated with workspace
  string workspacePassword = 3; // workspace user password
  string workspaceObjectName = 4; // resulting object name actually stored in backend
}
```

Reply:
- tohle je PROJECT handler, tady je to potreba.
- samotny WORKSPACE handler uz zadny dalsi granty nema.

Reply:
pridany testy na kontrolu prav